profiles/foot.lua

```diff
@@ -160,6 +160,9 @@ function way_function (way, result)
   if "roundabout" == junction then
     result.roundabout = true
   end
+  if "circular" == junction then
+    result.circular = false
```

Comment:
Okay, then maybe set both to false and add a comment. Otherwise this PR looks fine.

(Strictly speaking post-processing should handle this, right? as in emitting different instructions based on the duration between steps)